aggregator/src/main/resources/application.properties

```diff
@@ -0,0 +1,56 @@
+quarkus.http.port=8085
+
+# Change port for tests to avoid messing with local Kafka instance
+%test.quarkus.http.port=9085
+%test.quarkus.http.test-port=9085
```

Comment:
The ports should probably be different from the backend.

Reply:
Do you think this is important? I would say it's not. I mean if it would be important, then we would need to know every port of every project to be able to deploy the applications on OpenShift.

Reply:
It's not important for OpenShift deployment, but it is for local testing. We need to be able to run all apps (backend, aggregator...) at once on the same machine.